pkg/cluster/spec_test.go

```diff
@@ -131,6 +132,59 @@ func TestNewSpecValid(t *testing.T) {
 	validateSpecFromSimpleBundle(t, gotSpec)
 }
 
+func TestNewSpecFromClusterConfigTinkerbellValid(t *testing.T) {
```

Comment:
I only added this to meet the coverage minimun (because the test was missing), I didn't touch this functionality